cobc/ppparse.y

```diff
@@ -1368,6 +1373,29 @@ leap_second_directive:
 | OFF
 ;
 
+imp_directive:
+  /* GnuCOBOL 3.3 extension */
+  INCLUDE imp_include_sources
+  {
+	struct cb_text_list *p = $2;
+	while (p != NULL) {
+		fprintf (ppout, "#INCLUDE %s\n", p->text);
+		p = p->next;
+	}
+  }
+;
```

Comment:
please add error handling here,, something like a variant `errror` where you reset the error; this should help that the later parser does not output "missing program-id" (see your test case)